src/Elastic.Markdown/Myst/CodeBlocks/EnhancedCodeBlockHtmlRenderer.cs

```diff
@@ -107,6 +107,22 @@ private static int CountIndentation(StringSlice slice)
 		return indentCount;
 	}
 
+	private static bool IsCommentBlock(Block block)
+	{
+		if (block.GetType().Name == "CommentBlock")
+		{
+			return true;
+		}
+
+		if (block is HtmlBlock htmlBlock)
+		{
+			var content = htmlBlock.Lines.ToString();
+			return content.TrimStart().StartsWith('%');
+		}
```

Comment:
In which cases is this needed?

Out of curiousness I removed this part and ran the tests. And everything is still green.

Reply:
You're right! My first attempt at finding comments was to parse HTML blocks starting with `%` but that didn't work as comments create `CommentBlock`s not `HtmlBlock`s. This should be removed.

Reply:
```suggestion
```